roofit/batchcompute/CMakeLists.txt

```diff
@@ -63,3 +52,4 @@ if (ROOT_PLATFORM MATCHES "linux|macosx" AND CMAKE_SYSTEM_PROCESSOR MATCHES x86_
 
 endif()
 
+ROOT_INSTALL_HEADERS()
```

Comment:
Why is this needed?

Reply:
I saw that RooBatchCompute.h is used directly in RooFitCore/RoofitMore and since ROOT_LINKER_LIBRARY function is not installing headers, but only .so file I thought you may need it?

Reply:
Ah, I thought you have to list headers that should be installed. So this just installs all headers from `inc/`? That should be correct.

Reply:
yup, everything from `inc` :+1: